Cargo.toml

```
@@ -10,23 +10,23 @@ repository = "https://github.com/awslabs/coldsnap"
 keywords = ["AWS", "Amazon", "EBS", "snapshot"]
 
 [features]
-default = ["rusoto-native-tls"]
-rusoto-native-tls = ["rusoto_core/native-tls", "rusoto_ebs/native-tls", "rusoto_ec2/native-tls"]
-rusoto-rustls = ["rusoto_core/rustls", "rusoto_ebs/rustls", "rusoto_ec2/rustls"]
+default = ["aws-sdk-rust-native-tls"]
+aws-sdk-rust-native-tls = ["aws-config/native-tls", "aws-sdk-ebs/native-tls", "aws-sdk-ec2/native-tls"]
+aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-ebs/rustls", "aws-sdk-ec2/rustls"]
 
 [dependencies]
 argh = "0.1.7"
-tokio = { version = "1.8", features = ["fs", "io-util", "time"] }
+aws-config = "0.11.0"
+aws-sdk-ebs = "0.11.0"
+aws-sdk-ec2 = "0.11.0"
+aws-smithy-http = "0.41.0"
+aws-types = "0.11.0"
+tokio = { version = "~1.14", features = ["full"] }
```

Cargo.toml files that are used in libraries should not lock in a major.minor version because they can make it impossible for a client to build (if some other library requires some other minor version).

I also think that our libraries should not require a minimum tokio version unless there is some specific reason for it. If we wanted to lock to a different version in the Bottlerocket tools project with, e.g. `~1.8`, then the I think it fail because that would be lower than this one.

```suggestion
tokio = { version = "1", features = ["full"] }
```

---

My understanding is that we intentionally lock to the LTS version of tokio. Previously that had been 1.8.x, now it's 1.14.x. Is that not true?